js/app/src/Blocks.svelte

```diff
@@ -61,7 +62,7 @@
 			try {
 				d.frontend_fn = new AsyncFunction(
 					"__fn_args",
-					`let result = await (${d.js})(...__fn_args);
+					`console.log("$$$$$$$123: ", ...__fn_args);
```

Comment:
Is this intentional?

Reply:
I think this change broke the `js` functions. I can't run JS functions anymore -- for example, try `demo/blocks_js_methods/run.py`